main.go

```diff
@@ -1,4 +1,31 @@
 package main
 
+import (
+	"fmt"
+	"github.com/log-yarder/yarder/appender"
+	"github.com/log-yarder/yarder/storage"
+	"io/ioutil"
+	"log"
+	"os"
+)
+
+const (
+	maxEntriesPerChunk = 10
+)
+
 func main() {
+	tmpDir, err := ioutil.TempDir(os.TempDir(), "yarder-dev")
```

Comment:
you don't need to call `os.TempDir` at all, just pass `""` to `ioutil.TempDir`

Reply:
Done.